quickjspp.hpp

```diff
@@ -1868,4 +1888,48 @@ property_proxy<Key>::operator Value() const
 }
 }
 
+inline void Context::enqueueJob(std::function<void()> && job_fcn) {
+    // No need to free the new value. It will be freed automatically when
+    // the function is called.
+    JSValueConst job_val = newValue(std::move(job_fcn)).release();
+    JS_EnqueueJob(ctx, [](JSContext *ctx, int argc, JSValueConst *argv){
+        try
+        {
+            // use Value constructor rather than context.newValue to avoid retrieving the Context object.
+            Value{ctx, argv[0]}.as<std::function<void()>>()();
```

Comment:
Correct code would be
`Value{ctx, JS_DupValue(ctx, argv[0])}.as<std::function<void()>>()();` which also won't free argv[0]

Reply:
Ahhh, I see. Thanks!